snuba/settings/__init__.py

```diff
@@ -160,7 +165,7 @@
     "snuba-transactions-commit-log": "snuba-commit-log",
 }
 
-# Mapping of default Kafka topic name to broker config
+# Mapping of physical Kafka topic name to broker config
```

Comment:
Why change this? The `default` was very important -> it is supposed to be the name hardcoded in the Snuba codebase/dataset config and NOT the physical name that the user might override

Reply:
This make sense. I added a new `SLICED_KAFKA_BROKER_CONFIG` that will deal with sliced topics specifically, and have physical/custom names as keys